modules/mobile/app/workers/mobile/v0/vet360_linking_job.rb

```diff
@@ -28,7 +28,13 @@ def perform(uuid)
         user = IAMUser.find(uuid) || User.find(uuid)
         raise MissingUserError, uuid unless user
 
+        mobile_user = Mobile::User.find_or_create_by(icn: user.icn)
+        mobile_user.vet360_link_attempts += 1
```

Comment:
Could be worth having a `increment_attempts` method on the model for this.

Reply:
Ya that reads better